registry/server/routes/config.ts

```diff
@@ -8,7 +8,7 @@ router.get('/', async (req, res) => {
     const [apps, templates, routes] = await Promise.all([
         knex.select().from('apps'),
         knex.select().from('templates'),
-        knex.select('routes.id as routeId', '*')
```

Comment:
![image](https://user-images.githubusercontent.com/57893569/72344595-4f559280-36da-11ea-8155-6e901a5b5d8b.png)
Does this code work correctly with this change?

Reply:
Discussed